lib/main.dart

```diff
@@ -1,7 +1,15 @@
-import 'package:OutOfBounds/nve/nve-main-page.dart';
-import 'package:OutOfBounds/zht/zht-main-page.dart';
+//import 'package:OutOfBounds/nve/nve-main-page.dart';
+//import 'package:OutOfBounds/zht/zht-main-page.dart';
 import 'package:flutter/material.dart';
+import './pages/home_page.dart';
 
+
+void main(){
```

Comment:
Do NOT replace the main file content to stay compliant whith all the team contents. Please restore original file.